.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Tests
+on: [push, pull_request, create]
+jobs:
+  runTests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+      - name: Install dependencies
+        run: sudo apt-get install -y libevent-dev python3-pip python3-virtualenv
+      - name: Run Tests Tests
+        run: |
+          mvn package
```

Comment:
I'm not sure why I have to run `mvn package` while the travis script only ran `mvn test` but it seems that `s3proxy` isn't generated when I run `mvn package`

Reply:
I suspect that this is just some difference between Travis and Docker.  We might want to use `mvn verify` but let's follow up in a subsequent PR.